sonic_platform_base/sonic_ssd/ssd_generic.py

```diff
@@ -35,6 +35,11 @@ class SsdUtil(SsdBase):
     health = NOT_AVAILABLE
     ssd_info = NOT_AVAILABLE
     vendor_ssd_info = NOT_AVAILABLE
+    program_fail_count = NOT_AVAILABLE
```

Comment:
why class level variable?

Reply:
to be consistent with the existing statistics like health, temperature, ...
when a new SSD model will be supported, we need to implement these statistics with the new smart output.